src/Settings.elm

```diff
@@ -0,0 +1,132 @@
+module Settings exposing (..)
+
+import Accessibility.Aria as A11y exposing (..)
+import Html exposing (Html)
+import Html.Attributes as Attrs exposing (..)
+import Html.Events exposing (onClick)
+import Set exposing (Set)
+import SharedTypes exposing (..)
+
+
+viewSettingsForm : UserGroup -> Set String -> (String -> msg) -> Html msg
+viewSettingsForm userGroup isInputVisible onInputShow =
+    let
+        -- Extract settings data from the user group
+        settings =
+            userGroup.settingsData
+
+        -- Determine if the settings are inherited or not
+        isInherited =
+            case settings.inheritedFrom of
+                Just "Null" ->
```

Comment:
This feels a bit weird to me, I wonder where this `Null` comes from but probably it should be [catched in parsing](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/).

Reply:
What a great article @kutyel ! I agree, @kraklin in its current form it's not working, I must have changed something while refactoring and forgot to test for it, this was how it used to be and it worked perfectly.

```
isInherited =
            case settings.inheritedFrom of
                Nothing ->
                    False

                _ ->
                    True
```

and the fields didn't have a not 

`Attrs.disabled (not isInherited)`

This was an oversight from my end